recipes/gtk2/all/conanfile.py

```diff
@@ -0,0 +1,56 @@
+from conans import ConanFile, tools
+from conans.errors import ConanException
+
+
+class ConanGTK2(ConanFile):
+    name = "gtk2"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "LGPL-2.1-or-later"
+    homepage = "https://www.gtk.org"
+    description = "A free and open-source cross-platform widget toolkit for creating graphical user interfaces"
+    settings = {"os": "Linux"}
+    topics = ("gui", "widget", "graphical")
+    provides = "gtk"
+
+    def package_id(self):
+        self.info.header_only()
```

Comment:
I would suggest not to use ``header_only()``, but explicitly removing settings, options, and requirements, because this is not a header only library. Maybe we need another helper in Conan, but this reads weird.

Reply:
Done.